github/repos_pages.go

```diff
@@ -93,6 +93,25 @@ func (s *RepositoriesService) GetLatestPagesBuild(owner string, repo string) (*P
 	return build, resp, err
 }
 
+// GetPageBuild fetches the specific build information for a GitHub pages site.
+//
+// GitHub API docs: https://developer.github.com/v3/repos/pages/#list-a-specific-pages-build
+func (s *RepositoriesService) GetPageBuild(owner string, repo string, ID int) (*PagesBuild, *Response, error) {
+	u := fmt.Sprintf("repos/%v/%v/pages/builds/%v", owner, repo, ID)
+	req, err := s.client.NewRequest("GET", u, nil)
+	if err != nil {
+		return nil, nil, err
+	}
+
```

Comment:
I tweaked your command slightly:
> curl -i "https://api.github.com/repos/github-archive/developer.github.com/pages/builds" \
>      -H 'Authorization: token $TOKEN' \
>      -H "Accept: application/vnd.github.mister-fantastic-preview+json"

and now I get:
> HTTP/1.1 401 Unauthorized
> ...
> {
>   "message": "Bad credentials",
>   "documentation_url": "https://developer.github.com/v3"
> }

So I'm thinking this is an ownership thing.  I'll try creating a pages site.